enterprise/server/remote_execution/runner/runner.go

```diff
@@ -1093,7 +1093,7 @@ func (p *pool) newContainerImpl(ctx context.Context, props *platform.Properties,
 			DockerClient:           p.dockerClient,
 			ActionWorkingDirectory: p.hostBuildRoot(),
 			NumCPUs:                int64(math.Max(1.0, float64(sizeEstimate.GetEstimatedMilliCpu())/1000)),
-			MemSizeMB:              int64(math.Max(1.0, float64(sizeEstimate.GetEstimatedMemoryBytes())/1e6)),
+			MemSizeMB:              int64(math.Max(200.0, float64(sizeEstimate.GetEstimatedMemoryBytes())/1e6)),
```

Comment:
nit: We should make this `200.0` a constant in `tasksize.go`, like `FirecrackerMinMemoryMB`

I think we should duplicate (\*) this change to `tasksize.go` (setting a minimum memory size for firecracker actions), that way the executor's local scheduler (`priority_task_scheduler.go`) gets a more accurate view of how much memory the task will take, and doesn't dequeue too many tasks at once (which all have a very low mem estimate in their TaskSize but actually wind up being assigned 200MB).

(\*) The reason to duplicate instead of move is that there might be a (rare) case where a task doesn't set `workload-isolation-type` but is assigned to an executor that is configured to run with firecracker isolation by default. `tasksize.go` won't be able to tell that the task is a firecracker task in that case.